.github/workflows/botonic-cli-tests.yml

```diff
@@ -1,6 +1,12 @@
 name: Botonic cli tests
 
-on: [push]
+on:
```

Comment:
You are not uploading the coverage for this package.
I see it isn't running any test (there isn't any `npm run test`).
I would prefer to also upload this coverage that will be 0%, but this will make visible that we don't have any testing for this package.
But this can go in a separate PR to get this merged